doc/api/api.raml

```diff
@@ -1,7 +1,7 @@
 #%RAML 1.0
 ---
 title: Metamaps
-version: v2
+version: v2.0
```

Comment:
add a minor version so once we publish to metamaps.cc, we can bump this to 2.1 if we add new (backwards-compatible) functionality